packages/terra-embedded-content-consumer/src/EmbeddedContentConsumer.jsx

```diff
@@ -53,9 +63,21 @@ class EmbeddedContentConsumer extends React.Component {
     (customEvents || []).forEach(e => this.addEventListener(e.key, e.handler));
   }
 
+  /**
+  * Triggers an event in the application. Assumes this consumer has been mounted.
+  * @param {string} event - The event name to trigger.
+  * @param {object} detail - The data context to send with the event.
+  */
+  trigger(event, detail) {
```

Comment:
if you pass back the xfc frame this isn't needed

Reply:
Agreed this would not be needed if the xfcFrame is passed when mounted. Follow the [above discussion](https://github.com/cerner/terra-core/pull/938#discussion_r145528274).